project/WORKSPACE

```diff
@@ -0,0 +1,32 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```

Comment:
Please add a header here too just in case.

Reply:
Done ✅